testsuite/pom.xml

```diff
@@ -8,6 +8,18 @@
         <relativePath>..</relativePath>
     </parent>
     <artifactId>testsuite</artifactId>
+    <!-- manage Vert.x gRPC server dependency -->
```

Comment:
We do not use quarkus bom intentionally to avoid any class path side effects, pls define vertx-grpc-server version via property as in the case of other deps

Reply:
It cannot be managed by dependabot which will cause us forgetting to bump again and again. Dependabot don't know what version of Vert.x is Quarkus using and IIRC Quarkus is behind upstream Vert.x. I think tests have these deps as parent, so there will be conflicts between what Vert.x version is used by Quarkus application that is tested and by our test.

> We do not use quarkus bom intentionally to avoid any class path side effects

Can you give me an example? I'd expect that dependency versions are exactly same as are used by tested app and it's just a dependency management. I don't understand why having test dependencies managed by Quarkus platform should cause issues. I think I run into some logger inconsistencies that I didn't want to play with, that's why I moved this bom down to the `testsuite` module where it is required.

Reply:
dependabot argument is valid one, I will think more about the options tmr. Hope this is not under time pressure.

Reply:
No, I have other tasks, take your time. What I din't realize was that if we don't manage Vert.x gRPC server and just have dedicated property for it in the test scope, than it shouldn't affect Quarkus application that is started below. They do share dependencies as this is parent, but I doubt we have Vert.x gRPC server anywhere there. I won't swear on this change though.

Reply:
Hmm, maybe Vert.x versions could clash, I think it is fine as long as Vert.x managed by BOM in POM of MP apps is used, as this code is very generic.

Reply:
about "class path side effects" ... I want to have controlled version of the libraries used for the testsuite infrastructure to avoid any (even highly unexpected) interference from the versions defined in the bom.

I run StartStopTest with different Quarkus versions and keeping the "infra" same is important (to me :)).

So I'm staying on "No Quarkus BOM" position for now.